packages/edit-site/lib/lib-font.browser.js

```diff
@@ -468,7 +468,7 @@ class Woff2TableDirectoryEntry {
 		const transformVersion = ( this.transformVersion =
 			( this.flags & 192 ) >> 6 );
 		let hasTransforms = transformVersion !== 0;
-		if ( this.tag === `glyf` || this.tag === `loca` ) {
+		if ( this.tag === `glyf` || this.tag === `local` ) {
```

Comment:
This file is a copy of an external library, so I don't think any modifications are needed.